Makefile

```diff
@@ -364,10 +364,11 @@ deploy-onprem:
 	# Format: ip:hostPort:containerPort | ip::containerPort | hostPort:containerPort | containerPort
 	podman pod create --name assisted-installer -p 5432:5432,8000:8000,8090:8090,8080:8080
 	podman run -dt --pod assisted-installer --env-file onprem-environment --pull always --name db quay.io/ocpmetal/postgresql-12-centos7
-	podman run -dt --pod assisted-installer --env-file onprem-environment --pull always -v $(PWD)/deploy/ui/nginx.conf:/opt/bitnami/nginx/conf/server_blocks/nginx.conf:z --name ui quay.io/ocpmetal/ocp-metal-ui:latest
-	podman run -dt --pod assisted-installer --env-file onprem-environment ${PODMAN_PULL_FLAG} --env DUMMY_IGNITION=$(DUMMY_IGNITION) \
-		--restart always --name installer $(SERVICE)
-	./hack/retry.sh 90 2 "curl http://127.0.0.1:8090/ready"
+	podman run -dt --pod assisted-installer --env-file onprem-environment --pull always -v $(PWD)/deploy/ui/nginx.conf:/opt/bitnami/nginx/conf/server_blocks/nginx.conf:z --name ui quay.io/edge-infrastructure/assisted-installer-ui:latest
+	podman run -dt --pod assisted-installer --env-file onprem-environment --pull always --restart always --name image-service quay.io/edge-infrastructure/assisted-image-service:latest
+	podman run -dt --pod assisted-installer --env-file onprem-environment ${PODMAN_PULL_FLAG} --env DUMMY_IGNITION=$(DUMMY_IGNITION) --restart always --name installer $(SERVICE)
+	./hack/retry.sh 90 2 "curl -f http://127.0.0.1:8090/ready"
+	./hack/retry.sh 60 10 "curl -f http://127.0.0.1:8888/health"
```

Comment:
nit: this seems unrelated to the original PR
although I don't think it will do any harm

Reply:
This was the merge conflict in the Makefile